### PR TITLE
add basic nav bar to main menu

### PR DESCRIPTION
**Pull request by luzivaltic**

### CODE REVIEW

File: app/components/FlashCard.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { Card } from "@mui/material";
+
+export const FlashCard = () => {
+    return (
+        <>
+            <h1>Lalala</h1>
```
Comment (nolannguyen1212) [inline app/components/FlashCard.tsx:6]: Update images only if you want some demo, remove test code

File: app/components/MainMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { Fragment } from "react";
+
+export const MainMenu = () => {
+  return (
+    <div className="main-menu">
+
```
Comment (natalieconan) [inline app/components/MainMenu.tsx:6]: why main is empty and it still appears in git commit ?

File: app/components/auth/LoginForm.tsx
Comment (nolannguyen1212) [inline app/components/auth/LoginForm.tsx:1]: LoginForm is recommitted ?

File: app/components/auth/SignUpForm.tsx
Comment (nolannguyen1212) [inline app/components/auth/SignUpForm.tsx:1]: same here ?

File: app/pages/api/login/route.ts
Comment (nolannguyen1212) [inline app/pages/api/login/route.ts:1]: cool!

File: package.json
Comment (nolannguyen1212) [inline package.json:1]: why did @mui/icons-material need to update version ?

File: prisma/seed.ts
Comment (nolannguyen1212) [inline prisma/seed.ts:1]: ok!

**Review by william1nguyen — CHANGES_REQUESTED** [review]
Some files are recommitted ! Fix it!
Don't forget to remote test code!

File: package.json
Comment (nolannguyen1212) [reply package.json:1]: Screen Shot 2023-11-18 at 20 26 10
I've already updated @mui/icons-material in the develop branch

**Pull request closed without merging**